src/GuppyDataExplorer/ExplorerVisualization/index.jsx

```diff
@@ -81,18 +81,26 @@ class ExplorerVisualization extends React.Component {
             isLocked={isComponentLocked}
           />
         </div>
-        <div className='guppy-explorer-visualization__summary-cards'>
-          <DataSummaryCardGroup summaryItems={chartData.countItems} connected />
-        </div>
-        <div className='guppy-explorer-visualization__charts'>
-          <SummaryChartGroup
-            summaries={chartData.summaries}
-            lockMessage={lockMessage}
-            barChartColor={barChartColor}
-            useCustomizedColorMap={!!components.categorical9Colors}
-            customizedColorMap={components.categorical9Colors || []}
-          />
-        </div>
+        {
+          chartData.countItems.length > 0 && (
+            <div className='guppy-explorer-visualization__summary-cards'>
+              <DataSummaryCardGroup summaryItems={chartData.countItems} connected />
+            </div>
+          )
+        }
+        {
+          chartData.summaries.length > 0 && (
+            <div className='guppy-explorer-visualization__charts'>
+              <SummaryChartGroup
```

Comment:
Can we make more spaces between bar charts? Right now the tooltip is overlapping with the chart below

Reply:
I think thats fine - the tool tip could have a ton of data and we can't account for that so I just made the z-index higher so its the top most element